### PR TITLE
Change config file for each handler separately (#18)

### PR DESCRIPTION
**Pull request by ZeroRin**

### CODE REVIEW

File: tests/test_edit_log_path.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from subprocess import run
+
+__author__ = "ZeroRin"
+log = Path.cwd()/'logs/log.txt'
+modified_log =  Path.cwd()/'logs/log.log'
+
+def test_edit_log_path():
+    config_file = Path("../logger_tt/log_config.json")
+    config = json.loads(config_file.read_text())
+    config['handlers']['urllib_file_handler'] = config['handlers']['error_file_handler']
```
Comment (Dragon2fly) [inline tests/test_edit_log_path.py:14]: Please add a comment line so that we would know immediately what this line does 6 months later.

**Review by Dragon2fly — CHANGES_REQUESTED** [review]
Please exclude/remove `__pycache__` folder, change the annotation and add a little comment.

**Comment by ZeroRin** [comment]
Actually what I did was that I removed the `__pycache__` directory which has been there 2 years ago, probably before you add `__pycache__` into `.gitignore`.

Added comments as requested

**Review by Dragon2fly — APPROVED** [review]
Thank you. I'll remove the __pycache__ folder later.

**Pull request merged**